src/Tgstation.Server.Host/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove please, and maybe add .pubxml to the .gitigore

Reply:
I never made a publish so no idea how this happened